tests/resources/dockerfiles/Dockerfile.1-37-Docker-USER-UID:GID-2000:2000

```diff
@@ -0,0 +1,3 @@
+from busybox:latest
+user 2000:2000
```

Comment:
Please add another image with not exist user specified in the USER command, for example, notexist, notexist:notexist.
To test the not existed user will cause error during docker run.